### DISCUSSION TITLE
Debian VPS security updates, safe to apply ?

### DISCUSSION THREAD

**Discussion by ghost** [Ysurac/openmptcprouter#2102] [General]
Is it safe to apply security updates to a Debian VPS ? By "safe", I mean not likely to break the OMR configuration or functionality.

**Comment by Ysurac** [comment]
Yes, no problem. Custom release have "omr" in the name. You shouldn't have any update of them for now.

**Comment by ghost** [comment]
I just tried it, and it looks like it would _downgrade_ libjson-c3

Perhaps some other package required a newer version of it ?

**Reply by Ysurac** [reply]
Should not be a problem, if you have problem after that you can run the VPS install script again, it will keep key and fix any issue.
